jax/_src/numpy/lax_numpy.py

```diff
@@ -320,11 +321,43 @@ def _convert_and_clip_integer(val: ArrayLike, dtype: DType) -> Array:
   return clip(val, min_val, max_val).astype(dtype)
 
 
-@util.implements(np.load, update_doc=False)
-def load(*args: Any, **kwargs: Any) -> Array:
+def load(file: IO[bytes] | str | os.PathLike[Any], *args: Any, **kwargs: Any) -> Array:
```

Comment:
OOC can we use `os.PathLike[str]` here or do we want to allow both `str` and `bytes`?

Reply:
I'm honestly not sure, I thought `PathLike[Any]` would be the safest annotation here.